### DISCUSSION TITLE
Duda con router-view

### DISCUSSION THREAD

**Discussion by Desbarrancadero** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#518] [Q&A]
Hola, tengo una duda. No puedo desplegar el contenido de mi pagina pero si puedo ver el header y el footer en mi pagina.

Gracias
![pantalla app](https://user-images.githubusercontent.com/99223061/160061959-a3806804-2129-4078-970a-8d7183ea3f8c.PNG)

**Comment by OmarVenturaP** [comment]
si estas usando VUEX entonces puedes intentar usar "< router-view/ >" quizas eso queda. De todas maneras puedes ver en el "router/index.js"

**Reply by Desbarrancadero** [reply]
h onestamente estoy superperdido con eso. Como podria saber si uso o no vuex? gracias por el apoyo

**Reply by OmarVenturaP** [reply]
Si te sale el about entonces es VUEX, si solo tienes el HOMEVIEW entoces es VUE

**Comment by MauroMontan** [comment]
Por favor, podrias adjuntar captura de tu archivo de rutas (router/index.js), y de tu main.js, es para saber si se encuentra bien la implementación del router

**Reply by OmarVenturaP** [reply]
![image](https://user-images.githubusercontent.com/99059639/160068744-8dbe366c-262f-42a1-baf9-17673df58b8f.png)

Ese es mi archivo App.vue

**Reply by OmarVenturaP** [reply]
no tuve que meterle otra ruta en ese archivo. Solo en Home Vuex es que le ingrese el Footer y en el nav que viene por defecto nada mas lo estilice.

**Comment by Desbarrancadero** [comment]
Esto es mi main y mi index de routhe
![main](https://user-images.githubusercontent.com/99223061/160220242-3fe81b17-95fb-4427-a9ea-7787309d49ba.PNG)


gracias 
![router](https://user-images.githubusercontent.com/99223061/160220241-250fc69b-452a-4e2b-973b-60509b580b10.PNG)
r
